index.html

```diff
@@ -441,12 +442,19 @@ <h2 id="getStarted">Get started</h2>
         </header>
         <form action="https://formspree.io/f/xjvdryln" method="POST" id="contactForm">
             <div id="inlineInputs">
-                <input class="formInput" type="text" placeholder="Your name" maxlength="30" name="Your Name"
-                    id="yourName" required>
-                <input class="formInput" type="email" placeholder="Your email" name="Your email" id="yourEmail"
-                    pattern="[a-zA-Z0-9_]+([.][a-zA-Z0-9_]+)*@[a-zA-Z0-9_]+([.][a-zA-Z0-9_]+)*[.][a-zA-Z]{1,5}"
-                    required>
+                <div class="shortInputs" id="firstInput">
+                    <label class="formLabels" for="yourName">Your Name</label>
+                    <input class="formInput" type="text" placeholder="Your name" maxlength="30" name="Your Name"
+                        id="yourName" required>
+                </div>
+                <div class="shortInputs" id="secondInput">
+                    <label class="formLabels" for="yourName">Your email</label>
+                    <input class="formInput" type="email" placeholder="Your email" name="Your email" id="yourEmail"
+                        pattern="[a-zA-Z0-9_]+([.][a-zA-Z0-9_]+)*@[a-zA-Z0-9_]+([.][a-zA-Z0-9_]+)*[.][a-zA-Z]{1,5}"
+                        required>
+                </div>
```

Comment:
Edited the `HTML` structure and also the `CSS` to include `label` tag and make this contact form to be read louder compatible for accesibility